### ISSUE TITLE
[Package Request]: gurnec.HashCheckShellExtension -> idrassi.HashCheckShellExtension 

### ISSUE THREAD

**Issue by yjlin0224** [microsoft/winget-pkgs#99329]
### Package Requested

- [X] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [ ] I have performed a search and couldn't find this package.
- [X] I have checked that the installer for this package is not a script file
- [X] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: idrassi 
Publisher Url: https://github.com/idrassi/HashCheck
Package Name: HashCheckShellExtension 
Description: HashCheck Shell Extension
Package Version: 2.5.0
Installer URL: https://github.com/idrassi/HashCheck/releases/download/v2.5.0/HashCheckSetup-v2.5.0.1.exe

Current Package Identifier (for new versions):
please update `gurnec.HashCheckShellExtension` 2.4.0.55 to forked update 2.5.0 (Identifier maybe changed to `idrassi.HashCheckShellExtension `)

**Comment by JustMyGithub** [comment]
Even one year later, the gurnec one is still the only HashCheck in Winget - why?

**Comment by zotabee** [comment]
Yep, still appearing as gurnec.HashCheckShellExtension in winget even though I only have idrassi/HashCheck installed.

```
PS C:\Users\Me> winget list HashCheckShellExtension
Name                      Id                             Version Source
------------------------------------------------------------------------
HashCheck Shell Extension gurnec.HashCheckShellExtension 2.6.1.0 winget
```
